notebook/templates/tree.html

```diff
@@ -19,7 +19,7 @@
       <ul id="tabs" class="nav nav-tabs">
         <li class="active"><a href="#notebooks" data-toggle="tab">Files</a></li>
         <li><a href="#running" data-toggle="tab">Running</a></li>
-        <li><a href="#clusters" data-toggle="tab">Clusters</a></li>
+        <li><a href="#clusters" data-toggle="tab" {% block clusters_tab_attributes %}{% endblock %} >Clusters</a></li>
```

Comment:
It doesn't make much sense to do this on clusters, but not the other tabs. Perhaps there should be a `tabs` block?